authorization/index.md

```diff
@@ -443,6 +445,26 @@ keys:
       <td>Scope of access authorized. Note that this can be different from the scopes requested by the app.</td>
     </tr>
     <tr>
+      <td><code>host</code></td>
```

Comment:
This doesn't look right to me, @bakerdb. The response to a token request is a JSON object with keys demonstrated in the example below. Not a JWT with the keys you've listed here. Where are these claims coming from?